model/request/ProctorioExamUrlFactory.php

```diff
@@ -28,20 +28,38 @@
 class ProctorioExamUrlFactory extends ConfigurableService
 {
     public const OPTION_BASE_URL = 'base_url';
+    public const DELIVERY_EXECUTION_TAKE_URI = '/taoDelivery/DeliveryServer/runDeliveryExecution';
+    public const DELIVERY_EXECUTION_END_URI = '/taoDelivery/DeliveryServer/index';
+    public const DELIVERY_EXECUTION_START_URI = '/remoteProctoring';
+    public const ANY_CHARACTER_PATTERN = '.*';
```

Comment:
```suggestion
    public const DELIVERY_EXECUTION_TAKE_URI = '/taoDelivery/DeliveryServer/runDeliveryExecution';
    private const DELIVERY_EXECUTION_END_URI = '/taoDelivery/DeliveryServer/index';
    private const DELIVERY_EXECUTION_START_URI = '/remoteProctoring';
    private const ANY_CHARACTER_PATTERN = '.*';
```
The constants are only used internally.

Reply:
done !